### PR TITLE
Add Fall 2022 shows

### PR DESCRIPTION
**Pull request by noogai15**
Added a couple of upcoming anime

### CODE REVIEW

**Comment by reconman** [comment]
Why did you close this PR?

**Comment by noogai15** [comment]
Idk if anyone was gonna take a look at it so I was like🤷 but I can reopen

**Comment by reconman** [comment]
Better leave a comment next time. Sometimes life gets in the way and then you forget about some PRs and issues.

File: series-tvdb.en.yaml [hunk 1]
```diff
@@ -208,6 +215,11 @@ entries:
       - season: 2
         anilist-id: 20719
 
+  - title: "Chainsaw Man"
```
Comment (reconman) [inline series-tvdb.en.yaml:218]: The entry is not needed if there are no synonyms and it's a single season show.

File: series-tvdb.en.yaml [hunk 2]
```diff
@@ -174,6 +174,13 @@ entries:
       - season: 2
         anilist-id: 21861
 
+  - title: "Blue Lock"
```
Comment (reconman) [inline series-tvdb.en.yaml:177]: Is already covered by Anilist Romaji + English title, no mapping needed.

**Pull request merged**